salt/orch/kubernetes.sls

```diff
@@ -67,7 +67,7 @@ etcd_nodes_setup:
     - tgt: 'roles:etcd'
     - tgt_type: grain
     - highstate: True
-    - concurrent: True
+    - batch: 1
```

Comment:
When we start doing that we will have to do a lot of updates anyway - this is a known good setting for a large amount of machines.